### PR TITLE
Add yamllint for Openshift dir

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by beav** [comment]
has conflict but looks good, acking

File: openshift/templates/drift-backend.bc.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     app: drift-backend
   name: drift-backend
-  namespace: {{ project }}
+  namespace: "{{ project }}"
```
Comment (beav) [inline openshift/templates/drift-backend.bc.yaml]: I didn't realize quoting the jinja templates allow it to pass yamllint. I'll add this back for the openapi specs after this PR is merged.

Comment (beav) [reply openshift/templates/drift-backend.bc.yaml]: @ehelms looks like this is getting picked up by your check already:

```
$ yamllint drift/openapi/api.spec.yaml
drift/openapi/api.spec.yaml
  12:12     error    too many spaces inside braces  (braces)
  12:24     error    too many spaces inside braces  (braces)
  12:27     error    syntax error: expected <block end>, but found '{'
  12:29     error    too many spaces inside braces  (braces)
  12:38     error    too many spaces inside braces  (braces)
The command "yamllint drift/openapi/api.spec.yaml" exited with 1.
```

I'm guessing it just needs a handful of `"` thrown in on line 12.

Comment (ehelms) [reply openshift/templates/drift-backend.bc.yaml]: I had to update that in the spec for the tests to pass in this PR.

File: openshift/templates/drift-backend.dc.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ spec:
               value: /r/insights/platform/
             - name: APP_NAME
               value: drift
-          image: {{ project }}/drift-backend:latest
+          image: "{{ project }}/drift-backend:latest"
```
Comment (beav) [inline openshift/templates/drift-backend.dc.yaml]: :+1: thanks

**Pull request merged**